[13.0] helpdesk_mgmt: add text input in ticket list searchbar

luffah opened the pull request:
See https://github.com/OCA/helpdesk/issues/64 as original issue.
See https://github.com/OCA/helpdesk/pull/69 for v11.0.
See https://github.com/OCA/helpdesk/pull/141 for v12.0.

This PR add search bar text input to portal ticket list.
Searchable fields are : name; description; user; category; (optionally message content)
2 meta fields extend searchable fields : All (lookup all searchable fields); Content (name + description + optionally message content) 

Searching in message content is enabled by installing the module [base_search_mail_content](https://github.com/OCA/social/tree/13.0/base_search_mail_content).

cramos84 committed:
[ADD] New module helpdesk.project

DarioLodeiros committed:
[ADD] Unit Tests

oca-travis committed:
[UPD] Update helpdesk_mgmt_project.pot

OCA-git-bot committed:
[UPD] README.rst

dalonsod committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_project
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_project/es/

luffah committed:
[IMP] helpdesk_mgmt: add text input in ticket list searchbar

ygol committed:
Update __manifest__.py

fix typo

OCA-git-bot committed:
helpdesk_type 13.0.1.0.1

ypapouin committed:
Added translation using Weblate (French)

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (9 of 9 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_project
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_project/fr/

marcelsavegnago committed:
[13.0][IMP] readme.rst: Add addons table and translate status

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 88.9% (8 of 9 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_project
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_project/pt/

dalonsod committed:
[ADD] helpdesk_mgmt_timesheet: original files from PR #121

dalonsod committed:
[FIX] helpdesk_mgmt_project: missing context for task creation

DarioLodeiros committed:
[FIX] View Ticket clickable status bar progress

#153 Mantenaince tasks

DarioLodeiros committed:
[IMP] Prettier xml

DarioLodeiros committed:
[IMP] Prettier

OCA-git-bot committed:
helpdesk_mgmt_project 12.0.1.1.0

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (18 of 18 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_type
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_type/es_AR/

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (15 of 15 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_rating
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_rating/es_AR/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (14 of 14 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_motive
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_motive/es_AR/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (195 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/es_AR/

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (9 of 9 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_project
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_project/es_AR/

Commit:
Added translation using Weblate (Italian)

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (9 of 9 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_project
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_project/it/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/

dalonsod committed:
[IMP] helpdesk_mgmt_project: black, isort, prettier

dalonsod committed:
[MIG] helpdesk_mgmt_project: Migration to 13.0

oca-travis committed:
[UPD] Update helpdesk_mgmt_project.pot

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
helpdesk_mgmt_timesheet 12.0.1.1.0

Commit:
[IMP] helpdesk_mgmt_timesheet: black, isort

Commit:
[MIG] helpdesk_mgmt_timesheet: Migration to 13.0

flachica commented on helpdesk_mgmt/controllers/myaccount.py:
```diff
@@ -27,14 +29,21 @@ def _helpdesk_ticket_check_access(self, ticket_id):
+    # FIXME module to migrate dans oca/social
+    def _message_content_field_exists(self):
+        base_search_module = request.env['ir.module.module'].sudo().search([
+            ('name', '=', 'base_search_mail_content')])
```
You must put this dependence in <repo_root>/oca_dependencies.txt

luffah replied:
added base_search_mail_content in oca_dependencies.txt

flachica commented:
If yoy run pre-commit run -a you can solve [this](https://travis-ci.com/github/OCA/helpdesk/jobs/360492181#L278)

Functional test OK

Thanks for your work

oca-travis committed:
[UPD] Update helpdesk_mgmt_timesheet.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_project
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_project/

dalonsod committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (20 of 20 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet/es/

OCA-git-bot committed:
[UPD] addons table in README.md

GorkaDigital5 committed:
[IMP] Improved Data Output in Team tree view

[IMP] helpdesk_mgmt - Requested changes for performance improval.

dalonsod committed:
[IMP] helpdesk_mgmt: prepare ticket number generation

ypapouin committed:
[FIX] Allow base user to access partner data without being in a Helpdesk group

gdgellatly committed:
[FIX] Helpdesk Management Timesheet

Before this commit, in any view with ticket_id present but unused the onchange would be called and set project and task to False. It was found while trying to integrate project_timesheet_time_control with helpdesk_mgmt_timesheet, where this onchange overrides the results of default_get.

After this commit, if a helpdesk ticket is not present when calling onchange, project and task ids are not reset to False.

gdgellatly committed:
[NEW] Helpdesk_mgmt_timesheet_time_control

This new module extends the integration between helpdesk and timeheets. If project_timeshet_time_control and helpdesk_mgmt_timesheet are installed this module will autoinstall.

Fixups from review Co-authored-by: Jairo Llopis <Yajo@users.noreply.github.com>

OCA-git-bot committed:
helpdesk_mgmt_timesheet 13.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[IMP] pre-commit: pin node version [ci skip]

lmiguens-solvos committed:
[ADD] helpdesk_mgmt_partner_sequence

sbidoul committed:
Use prettier/pre-commit for compatibility with pre-commit 2.8 [ci skip]

sbidoul committed:
update dotfiles [ci skip]

sbidoul committed:
Upgrade pylint-odoo for 13.0 [ci skip]

This is apprently necessary for the
development status checks to work correctly.

QaidjoharBarbhaya committed:
[FIX] helpdesk_mgmt: fix the following traceback

2020-11-13 13:04:21,899 23323 ERROR odoo odoo.sql_db: bad query: SELECT "res_partner".id FROM "res_partner" WHERE (("res_partner"."id" in (%s,%s))  AND  ("res_partner"."active" = %s)) ORDER BY "res_partner"."display_name"
ERROR: can't adapt type 'res.partner'
2020-11-13 13:04:21,900 23323 INFO odoo odoo.addons.fetchmail.models.fetchmail: Failed to process mail from pop server test.
Traceback (most recent call last):
  File "/home/adept-tech/odoo/odoo/addons/fetchmail/models/fetchmail.py", line 195, in fetch_mail
    res_id = MailThread.with_context(**additionnal_context).message_process(server.object_id.model, message, save_original=server.original, strip_attachments=(not server.attach))
  File "/home/adept-tech/odoo/odoo/addons/mail/models/mail_thread.py", line 1138, in message_process
    thread_id = self._message_route_process(message, msg_dict, routes)
  File "/home/adept-tech/odoo/odoo/addons/mass_mailing/models/mail_thread.py", line 28, in _message_route_process
    return super(MailThread, self)._message_route_process(message, message_dict, routes)
  File "/home/adept-tech/odoo/odoo/addons/mail/models/mail_thread.py", line 1052, in _message_route_process
    thread = ModelCtx.message_new(message_dict, custom_values)
  File "/home/adept-tech/odoo/extra-addons/helpdesk_oca/helpdesk_mgmt/models/helpdesk_ticket.py", line 242, in message_new
    ticket.message_subscribe(partner_ids)
  File "/home/adept-tech/odoo/odoo/addons/mail/models/mail_thread.py", line 2740, in message_subscribe
    partner_ids = self.env['res.partner'].sudo().search([('id', 'in', partner_ids), ('active', '=', True)]).ids
  File "/home/adept-tech/odoo/odoo/odoo/models.py", line 1643, in search
    res = self._search(args, offset=offset, limit=limit, order=order, count=count)
  File "/home/adept-tech/odoo/odoo/odoo/addons/base/models/res_partner.py", line 710, in _search
    count=count, access_rights_uid=access_rights_uid)
  File "/home/adept-tech/odoo/odoo/odoo/models.py", line 4460, in _search
    self._cr.execute(query_str, where_clause_params)
  File "/home/adept-tech/odoo/odoo/odoo/sql_db.py", line 173, in wrapper
    return f(self, *args, **kwargs)
  File "/home/adept-tech/odoo/odoo/odoo/sql_db.py", line 250, in execute
    res = self._obj.execute(query, params)
psycopg2.ProgrammingError: can't adapt type 'res.partner'

DarioLodeiros committed:
[FIX] Development Status

Maintainer Task

OCA-git-bot committed:
[UPD] README.rst

DarioLodeiros committed:
[FIX] Development Status

Maintainer Task

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.1.0

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] addons table in README.md

luffah commented:
> If yoy run pre-commit run -a you can solve [this](https://travis-ci.com/github/OCA/helpdesk/jobs/360492181#L278)

yoy ? is it a tool ? ;)

emagdalenaC2i commented:
@luffah See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

flachica committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (195 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/es/

flachica committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_type
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_type/es/

flachica committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_motive
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_motive/es/

oca-travis committed:
[UPD] Update helpdesk_mgmt_timesheet_time_control.pot

oca-travis committed:
[UPD] Update helpdesk_mgmt_timesheet.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet/

Commit:
[13.0][FIX] Names in pages is necesary to inherit

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (195 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/es_AR/

teloconf committed:
[FIX] Greeting in the closed notification email

The greeting is directed to the user instead of the customer

oca-travis committed:
[UPD] Update helpdesk_mgmt.pot

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (195 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/es_AR/

dalonsod committed:
[IMP] helpdesk_mgmt_timesheet: filter tickets with activity on timesheets today

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

oca-travis committed:
[UPD] Update helpdesk_mgmt_partner_sequence.pot

dalonsod committed:
Added translation using Weblate (Spanish (es_ES, not used in OCA))

dalonsod committed:
Added translation using Weblate (Spanish)

ChristianSantamaria committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_partner_sequence
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_partner_sequence/es/

dalonsod committed:
[IMP] helpdesk_mgmt: search tickets from both number and name

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (6 of 6 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_partner_sequence
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_partner_sequence/pt_BR/

dalonsod committed:
[IMP] helpdesk_type: added search, grouping tickets by type, some demo data and proper icon

anasuarez1 committed:
Added translation using Weblate (Spanish)

anasuarez1 committed:
Added translation using Weblate (Spanish)

anasuarez1 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet_time_control
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet_time_control/es/

anasuarez1 committed:
Translated using Weblate (Spanish)

Currently translated at 98.9% (193 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/es/

ypapouin committed:
Added translation using Weblate (French)

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (6 of 6 strings)

Translation: helpdesk-12.0/helpdesk-12.0-helpdesk_mgmt_partner_sequence
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-12-0/helpdesk-12-0-helpdesk_mgmt_partner_sequence/fr/

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.2.1

OCA-git-bot committed:
[UPD] addons table in README.md

JordiBForgeFlow committed:
[FIX] helpdesk_mgmt: Rules on who can access to tickets from the Portal should
only be governed by the ir.rules.

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.2.2

OCA-git-bot committed:
[UPD] addons table in README.md

MateuGForgeFlow committed:
[FIX] helpdesk_mgmt: Fix helpesk typo

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.2.3

OCA-git-bot committed:
helpdesk_mgmt_project 13.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

tdombos committed:
Added translation using Weblate (Hungarian)

tdombos committed:
Translated using Weblate (Hungarian)

Currently translated at 93.8% (183 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/hu/

tdombos committed:
Added translation using Weblate (Hungarian)

tdombos committed:
Translated using Weblate (Hungarian)

Currently translated at 100.0% (9 of 9 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_project
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_project/hu/

tdombos committed:
Added translation using Weblate (Hungarian)

tdombos committed:
Translated using Weblate (Hungarian)

Currently translated at 95.0% (19 of 20 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet/hu/

tdombos committed:
Added translation using Weblate (Hungarian)

tdombos committed:
Translated using Weblate (Hungarian)

Currently translated at 83.3% (5 of 6 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet_time_control
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet_time_control/hu/

tdombos committed:
Translated using Weblate (Hungarian)

Currently translated at 96.9% (189 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/hu/

tdombos committed:
Translated using Weblate (Hungarian)

Currently translated at 96.9% (189 of 195 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/hu/

LoisRForgeFlow committed:
[13.0][MIG] helpdesk_mgmt: add migration script for external ID renaming.

JordiBForgeFlow committed:
[helpdesk_type][fix] only reset team and user when the new type
does not match

OCA-git-bot committed:
helpdesk_type 13.0.1.0.2

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
helpdesk_mgmt_timesheet 13.0.1.0.2

OCA-git-bot committed:
[UPD] addons table in README.md

oca-travis committed:
[UPD] Update helpdesk_mgmt_timesheet.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet/

manuelregidor committed:
[13.0][FIX]helpdesk_mgmt_timesheet: default_project not a field in helpdesk.ticket.team

ChristianSantamaria committed:
[IMP] helpdesk_mgmt_partner_sequence: black, isort, prettier

ChristianSantamaria committed:
[MIG] helpdesk_mgmt_partner_sequence: Migration to 13.0

OCA-git-bot committed:
helpdesk_mgmt_timesheet 13.0.1.0.3

OCA-git-bot committed:
[UPD] addons table in README.md

gdgellatly committed:
[FIX] 13.0 Helpdesk Management Dashboard Performance

Removes the useless related field todo_ticket_ids.

oca-travis committed:
[UPD] Update helpdesk_mgmt_partner_sequence.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
helpdesk_mgmt_partner_sequence 13.0.1.0.1

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_partner_sequence
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_partner_sequence/

OCA-git-bot committed:
helpdesk_mgmt_timesheet 13.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] addons table in README.md

oca-travis committed:
[UPD] Update helpdesk_mgmt.pot

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (198 of 198 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/es_AR/

oca-travis committed:
[UPD] Update helpdesk_mgmt.pot

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.4.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (199 of 199 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt/es_AR/

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

oca-travis committed:
[UPD] Update helpdesk_type.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
helpdesk_type 13.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_type
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_type/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (6 of 6 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet_time_control
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet_time_control/es_AR/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (6 of 6 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_partner_sequence
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_partner_sequence/es_AR/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (22 of 22 strings)

Translation: helpdesk-13.0/helpdesk-13.0-helpdesk_mgmt_timesheet
Translate-URL: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-helpdesk_mgmt_timesheet/es_AR/

francesco-ooops committed:
Added translation using Weblate (Italian)

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
helpdesk_mgmt 13.0.1.5.0

OCA-git-bot committed:
[UPD] addons table in README.md

HTPit committed:
Added translation using Weblate (Chinese (Traditional))

HTPit committed:
Added translation using Weblate (Chinese (Traditional))

luffah commented:
I was looking again on this PR.
If i understand well, it is blocked because a test is missing.

luffah committed:
[IMP] helpdesk_mgmt: add text input in ticket list searchbar

luffah commented:
merged with most recent changes…
oh crap, i shall redo a new PR…
